doc/Makefile

```diff
@@ -4,8 +4,6 @@
 # You can set these variables from the command line, and also
 # from the environment for the first two.
 
-# TODO: Support nitpicky mode in readthedocs
```

Comment:
Moved this comment in to `conf.py`; pretty sure we can enable nitpicky mode there.